pkgs/development/tools/asn2quickder/default.nix

```diff
@@ -1,30 +1,22 @@
-{ stdenv, fetchFromGitHub, python2Packages, makeWrapper }:
+{ stdenv, fetchFromGitHub, python2Packages, makeWrapper, cmake }:
 
-stdenv.mkDerivation rec {
+python2Packages.buildPythonPackage rec {
```

Comment:
Better use `python2Packages.callPackage` and take `buildPythonPackage` as well as all dependencies as arguments.

Reply:
I don't understand what you mean by that? I thought his was a very common way of doing this.
Do you have a recommended example where this is done elsewhere in nixpkgs.
I grepped but could not find something that seems to fit here ...

Reply:
In `all-packages.nix`, one can use `python2Packages.callPackage`. And in `default.nix` the first line should look something like
```
{ stdenv, fetchFromGitHub, python2Packages, makeWrapper, cmake, pytestrunner, pytest, six, pyparsing }:
```

Reply:
Actually, this should be `buildPythonApplication`, since it's not a library.

Reply:
@dotlambda: When I do that, for some reason it tries to create a wheel and cannot find six. So it doesn't build anymore.

It builds perfectly fine as is. Would it not be okay to leave it as is? I only did a version bump...

Reply:
Thanks. Done.